presto-main/src/test/java/com/facebook/presto/execution/MockRemoteTaskFactory.java

```diff
@@ -141,6 +141,11 @@ public MockRemoteTask createRemoteTask(
         return new MockRemoteTask(taskId, fragment, node.getNodeIdentifier(), executor, scheduledExecutor, initialSplits, partitionedSplitCountTracker);
     }
 
+    @Override
+    public void destroyExchangeSources(Set<URI> locationsToDestroy)
+    {
```

Comment:
I understand that you cannot throw an exception here. However, it's pretty dangerous silently doing nothing here.

If someone write a new test and it didn't work because of this, the amount of work it would require to fix this is going to be very high. I imagine it wouldn't be too hard to implement this. You would need a map. But otherwise, it shouldn't take more than 10 lines.